mmv1/third_party/terraform/resources/resource_storage_bucket_object.go

```diff
@@ -161,16 +163,52 @@ func resourceStorageBucketObject() *schema.Resource {
 				DiffSuppressFunc: compareCryptoKeyVersions,
 				Description:      `Resource name of the Cloud KMS key that will be used to encrypt the object. Overrides the object metadata's kmsKeyName value, if any.`,
 			},
+
+			"customer_encryption": {
```

Comment:
wdyt about encryption here? customer doesn't make as much sense if we aren't mirroring the rest api. If you have another name or want to leave it that is fine too 
```suggestion
			"encryption": {
```

Reply:
tbh I'd rather to keep this as is, this makes it at least semi-consitent with name CSEK. Also if we'd have just encryption this could suggest that unless this is set, object isn't encrypted at all, which isn't true.

Reply:
sounds good